cplusplus.json

```diff
@@ -953,5 +953,52 @@
+  {"id": "cpp.libphenom", "name": "libPhenom", "tech": ["app-arch"], "tags": ["eventing"], "repo": "facebook/libphenom",
+  "imports": ["phenom/"], "examples": ["https://github.com/facebook/libphenom/tree/master/examples"]},
+  {"id": "cpp.libsourcey", "name": "LibSourcey", "tech": ["networking"], "tags": ["evented"], "repo": "sourcey/libsourcey",
+  "imports": ["scy/"], "examples": ["https://sourcey.com/libsourcey/examples/"]},
+  {"id": "cpp.libu", "name": "LibU", "tech": ["utils"], "tags": ["misc"], "repo": "koanlogic/libu",
+  "imports": ["u/libu.h"], "examples": ["https://github.com/koanlogic/libu/tree/master/example"]},
+  {"id": "cpp.loki", "name": "Loki", "tech": ["app-arch"], "tags": ["design-patterns"], "repo": "https://sourceforge.net/p/loki-lib/code/HEAD/tree/trunk/",
+  "imports": ["loki/"], "examples": ["https://sourceforge.net/p/loki-lib/code/HEAD/tree/trunk/test/"]},
+  {"id": "cpp.mili", "name": "MiLi", "tech": ["app-arch"], "tags": ["collection", "misc"], "repo": "https://bitbucket.org/fudepan/mili/src",
+  "imports": ["mili/"], "examples": ["https://bitbucket.org/fudepan/mili/src/6ec9fce00b988a7759a80e97ada52533f89fd2fb/examples/?at=default"]},
+  {"id": "cpp.open-frameworks", "name": "openFrameworks", "tech": ["graphics-gaming"], "tags": ["creative-coding"], "repo": "openframeworks/openFrameworks",
+  "imports": ["ofMain.h", "of"], "examples": ["https://github.com/openframeworks/openFrameworks/tree/master/examples"]},
+  {"id": "cpp.root", "name": "ROOT", "tech": ["big-data"], "tags": ["analyzing", "visualization"], "repo": "root-project/root",
+  "imports": ["root/"], "examples": ["https://github.com/root-project/root/tree/master/tutorials"]},
+  {"id": "cpp.stxxl", "name": "STXXL", "tech": ["big-data"], "tags": ["stl-kind"], "repo": "stxxl/stxxl",
+  "imports": ["stxxl/"], "examples": ["https://github.com/stxxl/stxxl/tree/master/examples"]},
+  {"id": "cpp.tbox", "name": "tbox", "tech": ["app-arch"], "tags": ["cross-platform-clib"], "repo": "tboox/tbox",
+  "imports": ["tbox/tbox.h"], "examples": ["https://github.com/tboox/tbox/tree/master/src/demo"]},
+  {"id": "cpp.ultimatepp", "name": "Ultimate++", "tech": ["gui"], "tags": ["cross-platform"], "repo": "ultimatepp/mirror",
+  "imports": [], "examples": ["https://github.com/ultimatepp/mirror/tree/master/examples"]},
+  {"id": "cpp.ustl", "name": "uSTL", "tech": ["app-arch"], "tags": ["less-memory"], "repo": "msharov/ustl",
+  "imports": ["uspecial.h", "ualgo.h", "unumeric.h", "ofstream.h"], "examples": ["https://github.com/msharov/ustl/tree/master/test"]},
+  {"id": "cpp.wtl", "name": "WTL", "tech": ["gui"], "tags": ["windows"], "repo": "https://sourceforge.net/p/wtl/code/HEAD/tree/trunk/",
+  "imports": ["atl"], "examples": ["https://sourceforge.net/p/wtl/code/HEAD/tree/trunk/wtl/Samples/"]},
+  {"id": "cpp.yomm2", "name": "YOMM2", "tech": ["app-arch"], "tags": ["multiple-dispatch"], "repo": "jll63/yomm2",
```

Comment:
tags==open-methods maybe?

Reply:
I thought multiple-dispatch is more technical word

Reply:
i guess multiple-dispatch is fine, too.